lib/flight_job/outputs/info_job.rb

```diff
@@ -28,107 +28,99 @@
-      <%= pastel.green submit_outputs.first.call(model) %>
-      <%= pastel.blue.bold "Submit Stderr" -%><%= pastel.bold ':' %>
-      <%= pastel.green submit_outputs.last.call(model) %>
-      <% end -%>
-    ERB
-
-    register_attribute(header: 'ID') { |j| j.id }
-    register_attribute(header: 'Script ID') { |j| j.script_id }
-    register_attribute(header: 'Scheduler ID') { |j| j.scheduler_id }
-    register_attribute(header: 'State') { |j| j.state }
-
-    # Show a boolean in the "simplified" output, and the exit code in the verbose
-    # NOTE: There is a rendering issue of integers into the TSV output. Needs investigation
-    register_attribute(header: 'Submitted', verbose: false) { |j| j.submit_status == 0 }
-    register_attribute(header: 'Submit Status', verbose: true) { |j| j.submit_status.to_s }
-
-    register_attribute(header: 'Submitted at') do |job, verbose:|
-      if verbose
-        job.created_at
-      else
-        DateTime.rfc3339(job.created_at).strftime('%d/%m/%y %H:%M')
-      end
+  class Outputs::InfoJob < OutputMode::Formatters::Show
+    def initialize(job, submit: true, **opts)
+      @submit = submit
+      super(job, **opts)
+    end
+
+    def submit?
+      @submit ? true : false
     end
 
-    start_header = ->(job, verbose:) do
-      job.actual_start_time || verbose ? 'Started at' : 'Estimated Start'
+    def format_row(field, value, padding)
+      f = pastel.blue.bold field
+      s = pastel.bold ':'
+      v = pastel.green value
+      "#{padding}#{f}#{s} #{v}"
     end
-    register_attribute(header: start_header) do |job, verbose:|
-      if job.actual_start_time || verbose
-        Outputs.format_time(job.actual_start_time, verbose)
+
+    alias_method :job, :object
+
+    constructor do
+      template(<<~ERB) if interactive?
+        <% each(:default) do |value, padding:, field:| -%>
+        <%= format_row(field, value, padding) %>
+        <% end -%>
+        <%
+          submit_outputs = callables.select { |proc| proc.config[:section] == :submit }
+          if verbose? || submit?
+        -%>
+
+        <%= pastel.blue.bold "Submit Stdout" -%><%= pastel.bold ':' %>
+        <%= pastel.green format(job.submit_stdout) %>
+        <%= pastel.blue.bold "Submit Stderr" -%><%= pastel.bold ':' %>
+        <%= pastel.green format(job.submit_stderr) %>
+        <% end -%>
+      ERB
+
+      register(header: 'ID') { job.id }
+      register(header: 'Script ID') { job.script_id }
+      register(header: 'Scheduler ID') { job.scheduler_id }
+      register(header: 'State') { job.state }
+
+      # Show a boolean in the "simplified" output, and the exit code in the verbose
+      # NOTE: There is a rendering issue of integers into the TSV output. Needs investigation
+      if verbose?
+        register(header: 'Submit Status') { job.submit_status.to_s }
       else
-        Outputs.format_time(job.estimated_start_time, false)
+        register(header: 'Submitted') { job.submit_status == 0 }
       end
```

Comment:
It's really hard to follow what's happening form here on down.  There are 5 different conditionals branched on for a total of 6 branches.  Something like the following would be much clearer:

```ruby
constructor do
  if verbose?
    register_common
    # ... register other cols
  elsif interactive?
    register_common
    # ... register other cols
  else
    register_common
    # ... register other cols
  end
end
```

Reply:
Will look what I can do, however at best it will end up:

```
if verbose?
  ...
elsif interactive?
  ...
  job.actual_start_time ? register_actual_* : register_estimated_*
  job.actual_end_time ? register_actual_* : register_estimated_*
else
  ...
end
```